algebra/src/polynomial/field/ntt/inv.rs

```diff
@@ -0,0 +1,16 @@
+use num_traits::Inv;
+
+use crate::{reduce::ReduceInvAssign, Field, NttField};
+
+use super::FieldNttPolynomial;
+
+impl<F: NttField> Inv for FieldNttPolynomial<F> {
+    type Output = Self;
```

Comment:
I think the Output type should be Option<Self>, some polynomials do not have inversions.